### ISSUE TITLE
Flash5 Movie silently not starting (first frame visible)

### ISSUE THREAD

**Issue by andremichelle** [ruffle-rs/ruffle#7301]
### Describe the bug

Open http://basementmx.andre-michelle.com/
<img width="212" alt="image" src="https://user-images.githubusercontent.com/6459974/175899098-e69af11c-8f36-4c18-8b20-15e75a737b6d.png">
When looking at the network tab, I see that all swf have been loaded. Testing in Chrome. The self-hosted version does also not work correctly.

### Expected behavior

This button usually shows preloading and then acts as a "drop-right" menu with 8 audio icons. 

### Affected platform

Browser's extension

### Operating system

mac monterey

### Browser

Chrome 102

**Comment by n0samu** [comment]
I haven't found the specific issue, but @Toad06 pointed out that this makes heavy use of `onClipEvent`, some features of which are unimplemented in Ruffle, so that may be the direction to look into.

**Comment by Toad06** [comment]
Yes, I think the issue is due to `\frame 2\PlaceObject2 (3) Depth: 2 (loops)\onClipEvent(data)` in "mixer.swf": when the call to `loops.loadMovie(bank + "/loopset.swf");` succeeds, Ruffle should trigger this data event but this never happens. 

Depending on the feedbacks on #7311, I could try to make a fix.

**Comment by andremichelle** [comment]
That would be great. I was hoping to present this flash movie at WAC2022 as part of Flash audio history.

**Comment by n0samu** [comment]
If needed, there are definitely ways to get your site up and running for the demo using the original Flash Player! Feel free to join our Discord server or contact me at `nosamu#3385` on Discord for details.

**Comment by andremichelle** [comment]
Thanks. I sent you a short message on discord.

**Comment by Toad06** [comment]
So I tried to implement `onData` but realized there was another issue that I wasn't able to fix. I'm posting my observations below.

---

[Sample 1](https://github.com/ruffle-rs/ruffle/files/9039827/sample_1.zip)
In `\frame 2\PlaceObject2 (3) Depth: 2 (loops)`, there are two clip events: `data` and `enterFrame`. The latter one should fire continuously and trace "frame" but this doesn't happen in Ruffle.

[Sample 2](https://github.com/ruffle-rs/ruffle/files/9039834/sample_2.zip)
This is almost the same sample as above, but the order of the clip events in the tree has been reversed (so `enterFrame` comes before `data`). This time, Ruffle succeeds to fire the event and trace "frame".

---

https://github.com/ruffle-rs/ruffle/blob/49e8f4e939c6b0e388a707c33c48ea397be72109/core/src/display_object/movie_clip.rs#L1134-L1137

After this code, I added the following line: `if id == 3 { log::error!("{:?}", clip_actions); }`

This logs:
- Sample 1: `[]`
- Sample 2: `[ClipAction { events: ENTER_FRAME, key_code: None, action_data: [136, 2, 0, 0, 0, 150, 7, 0, 0, 102, 114, 97, 109, 101, 0, 38, 0] }]` 

In both cases, the data event is missing.

---

A possible reason would be that `ClipEventFlag::DATA` doesn't work in SWFv5 in Ruffle:
https://github.com/ruffle-rs/ruffle/blob/49e8f4e939c6b0e388a707c33c48ea397be72109/swf/src/types.rs#L617-L618

However, apart from that comment, I haven't found any evidence that DATA was version-gated.
